.github/actions/gitops-release/action.yaml

```diff
@@ -26,6 +34,12 @@ inputs:
   output-directory:
     required: true
     description: "Output directory, in the GitOps repo to place the output, usually a helm chart"
+  master-values-path:
+    description: "where to copy the master values from, either a branch or release"
+  gitops-name:
+    description: "the name of the app in the gitops repo"
+  ingress-path:
```

Comment:
Is this used?

Reply:
It's not, the canary script has the inputs rendered into it and then passed to the gitops release action. Have removed